src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx

```diff
@@ -213,6 +220,7 @@ const DateForm: React.FC<{
     <RowButtonWrapper isSplit={isSplit}>
       {isSplit && <StartDateRowLabel>{quoteDisplayName}</StartDateRowLabel>}
       <RowButton
+        disabled={currentInsurer && !startDate}
```

Comment:
Why is `Boolean` not needed here?

Reply:
Good question. I'm no sure 😅 
Either way, I'm gonna add it.